src/test/java/org/jboss/mjolnir/GitHubClientTest.java

```diff
@@ -0,0 +1,60 @@
+package org.jboss.mjolnir;
+
+
+import org.eclipse.egit.github.core.Team;
+import org.eclipse.egit.github.core.User;
+import org.eclipse.egit.github.core.client.GitHubClient;
+import org.jboss.mjolnir.githubclient.ExtendedTeamService;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Sample test class illustrating the use of GitHubClient.
+ *
+ * (Not to be run during build process.)
+ *
+ * @author Tomas Hofman (thofman@redhat.com)
+ */
+@Ignore
+public class GitHubClientTest {
+
+	private static String TOKEN = "408af3afceec32628057e6cc6c8eda738c4a8951";
```

Comment:
Bad indentation. I'm also not sure whether it is wise to put actual tokens in test code on a test org. Now anybody can mess that one up.